Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -44,16 +44,18 @@ protected override void Awake()
                         break;
                     }
 
-                    RegisterPointer(pointingSource);
+                    RegisterFocuser(pointingSource);
```

Comment:
I think don't think there's a strong reason to rename Pointers to Foucusers.

Reply:
We're not renaming them, we're splitting them into two distinct entities. Focusers can focus on a thing without a ray. Eg, by touching them, or whatever. Pointers are a kind of Focuser.

This change has to come sooner or later.

Reply:
Ah I see. Well why not just use the built in `Input.Selectable` to handle this kinda stuff?

Reply:
I don't see another place where we're registering focusers that arent' pointers, or is this in a later revision?

Reply:
It's in the latest revision. RegisterFocuser (line 172) now takes an actual IFocuser instead of a pointingSource. I haven't changed the registeredPointers array to a registeredFocusers array yet, so that naming might cause confusion.